components/server/ee/src/user/stripe-service.ts

```diff
@@ -117,7 +117,7 @@ export class StripeService {
 
     async cancelSubscription(subscriptionId: string): Promise<void> {
         await reportStripeOutcome("subscriptions_cancel", () => {
-            return this.getStripe().subscriptions.del(subscriptionId);
+            return this.getStripe().subscriptions.del(subscriptionId, { invoice_now: true });
```

Comment:
Do we also need to bring this change into the `stripeService.cancelSubscription()` on the usage component?

Reply:
Do you wanna add it in this PR or a follow-up?

Reply:
SGTM, unhold at will

Reply:
@easyCZ Would you have a direct link to the code location you meant by "the `stripeService.cancelSubscription()` on the usage component?"

I can't find the relevant code, and in fact we never seem to actually cancel Stripe subscriptions from the usage component? 🤔 (I only found [this function called CancelSubscription](https://github.com/gitpod-io/gitpod/blob/ee8354aa79e550f836e6d4f461d85fb59134e32b/components/usage/pkg/apiv1/billing.go#L337-L365), but it does not seem to call Stripe and only updates our internal models like the CostCenter).

Reply:
Also, when grepping the code, the only Go-side Stripe API methods we call are `Subscriptions.New` (1 occurrence) and `Subscriptions.Get` (1 occurrence). No `Subscriptions.Cancel` anywhere.